frontend/src/pages/InducteeSignUpPage/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Avatar, Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import SignUpForm from './components/SignUpForm';
+import styles from './styles';
+
+import { Card } from '@SharedComponents';
+import { PublicPageLayout } from '@SharedComponents/layouts';
+import HKN_TRIDENT_LOGO from '@Images/hkn-trident.png';
+import { createNewUser } from '@Services/AuthService';
+import {
+  doSignInWithEmailAndPassword,
+  doSendVerificationEmail,
+  doSignOut,
+} from '@Services/auth';
+
+const INITIAL_STATE = {};
+
+class InducteeSignUpPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { ...INITIAL_STATE };
+  }
+
+  handleSubmit = async (values, setSubmitting) => {
+    const {
+      email,
+      firstName,
+      lastName,
+      major,
+      gradYear,
+      password,
+      preferName,
+      pronoun,
+      customPronoun,
+      infoSession,
+      courseRequirement,
+      newsletter,
+    } = values;
+    const signupSubmission = {
+      email,
+      firstName,
+      lastName,
+      major,
+      password,
+      graduationYear: gradYear.toString(),
+      preferName,
+      pronoun,
+      customPronoun,
+      infoSession,
+      courseRequirement,
+      newsletter,
+    };
+
+    try {
+      await createNewUser(signupSubmission);
+    } catch {
+      console.log('Create new user failed');
+      setSubmitting(false);
+      return;
+    }
+
+    try {
+      await doSignInWithEmailAndPassword(email, password, false);
+    } catch {
+      console.log('Sign in failed');
+      setSubmitting(false);
+      return;
+    }
+
+    try {
+      await doSendVerificationEmail();
+    } catch {
+      console.log('Send verification email failed.');
+    }
+
+    await doSignOut();
+    setSubmitting(false);
+    alert('You have successfully signed up for an account.');
+  };
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <PublicPageLayout>
+        <Card className={classes.card}>
+          <Grid
+            container
+            className={classes.cardContent}
+            alignItems='center'
+            direction='column'
+            spacing={2}
+          >
+            <Grid item>
+              <Avatar className={classes.logo} src={HKN_TRIDENT_LOGO} />
+            </Grid>
+            <h2>INDUCTEE SIGN UP</h2>
+            <p> Congratulation! Description ... </p>
```

Comment:
What is this for?

Reply:
that's just the congrat block at the beginning. it's just a placeholder, haven't put the actual block of words yet